src/tabs/tab.jsx

```diff
@@ -126,13 +126,21 @@ const Tab = React.createClass({
 
     let iconElement;
     if (icon && React.isValidElement(icon)) {
+      let iconStyle = {
+        fontSize: 24,
+        marginBottom: (label) ? 5 : 0,
+        display: label ? 'block' : 'inline-block',
+        color: styles.root.color,
+      };
+      if (icon.type.displayName !== 'FontIcon' && label) {
```

Comment:
That's actually the original code (just re-factored so I can augment it with the margin styles), but I'll see if I can find a more generic solution with flexboxes.